### PR TITLE
Adds Vault password policy scenario

### PR DESCRIPTION
**Pull request by burtlo**
- No RabbitMQ based image exists within the core images. Attempted
  to install the package on Ubuntu 19.04 image does not work. I am
  including the setup scripts here but they do not work. An image
  will need to be found or created.
- Uses a custom-built Vault binary for 1.5 from master
  Vault v1.4.0 ('99564f52d0e570719050ab80326bb91c25188160+CHANGES')
- Language in intro and steps is not complete

### CODE REVIEW

File: vault-password-policies/courseBase.sh [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+# RabbitMQ: enable HTTP management port
+sudo rabbitmq-plugins enable rabbitmq_management
+
+# RabbitMQ: create user and add the administrator tag
+sudo rabbitmqctl add_user learn_vault hashicorp
+sudo rabbitmqctl set_user_tags learn_vault administrator
+
+# Vault: install custom-built Vault 1.5 binary
+wget https://lynn-vault-binaries.s3.us-east-2.amazonaws.com/vault
```
Comment (brianshumate) [inline vault-password-policies/courseBase.sh:14]: 🎉

File: vault-password-policies/intro.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+Vault can gen onfigurable password policies today and the impetus as to why we
+decided to do this. So, as you may know vault can generate dynamic usernames and
+passwords and we do that by randomly. Picking characters from a base 62
+character set right so when we decided to roll this out to certain customers.
+They came back and said, hey, this password that you're generating is to secure
+That means our older systems that we're connecting to like DB two systems have a
+very strict password character SEC requirement, they need A character password
+length of eight characters and has to be one uppercase one lowercase and one
+number one character only
```
Comment (brianshumate) [inline vault-password-policies/intro.md:9]: This intro text does not appear to be complete yet?

Comment (burtlo) [reply vault-password-policies/intro.md:9]: Yes, pushing the change now that the guide has gotten a review.

**Comment by burtlo** [comment]
6a48477 fixes the running of RabbitMQ 
The workstation now relies on the Hashistack image which will need the Vault 1.5 binary.

**Comment by burtlo** [comment]
Merged now that the HashiStack base image has been updated to include Vault 1.5 release candidate.

**Pull request merged**